src/router/routes/exportRoute.ts

```diff
@@ -59,17 +59,43 @@ export default class ExportRoute {
 
         // Export Job Status
         this.router.get(
-            '/\\$export/:jobId',
+            '/\\$export/:requesterUserId/:jobId',
```

Comment:
Hmm is this expected part of the spec?

Reply:
The FHIR spec didn't specify what the job status URL be, so this is up for us to decide. I'm including this so we can use it to authorized the user. We get the user's `userId` from their `accessToken` and compare it to the `requesterUserId` here to confirm that they are the same user. Given that they are authorized to get this job's status.

Reply:
Let's talk offline on a good solution